touchforms/formplayer/static/formplayer/script/test_helpers/jasmine-fixture.js

```diff
@@ -0,0 +1,433 @@
+/* jasmine-fixture - 1.2.2
```

Comment:
yea, does our deploy automatically compress? i usually like using the dev version in case i need to step through